nmrfx-core/src/main/java/org/nmrfx/chemistry/SpatialSet.java

```diff
@@ -624,8 +624,8 @@ public String toMMCifString(int iAtom, int iStruct) {
         //auth chain id 
         Object authChainID = atom.entity.getPropertyObject("authChainCode");
         //auth atom name
-        Object authAName = null;
-        
+        Object authAName = atom.getProperty("authAtomName");
```

Comment:
Retrieved this based on how the authAtomName was being set in MMcifReader which is where the preceding 3 objects were set. https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-core/src/main/java/org/nmrfx/chemistry/io/MMcifReader.java#L624